app/js/createUserObject.ts

```diff
@@ -132,6 +163,8 @@ async function createUsersObject() {
                 obj['name'] = user.name;
                 obj['email'] = user.email;
                 obj['score'] = result.score;
+                obj['categoryName'] = user.categoryName;
```

Comment:
these semi colons exist... so maybe make ALL the page semicolons. then by campsite rules it will be better than you found it..!